File: app/src/debug/java/org/pocketx/knell/KnellDebugApp.java
package org.pocketx.knell;

import org.pocketx.knell.base.KnellApp;

import timber.log.Timber;

public class KnellDebugApp extends KnellApp {

    @Override
    public void onCreate() {
        Timber.plant(new Timber.DebugTree());
        super.onCreate();
    }
}

File: app/src/debug/java/org/pocketx/knell/KnellDebugApp.kt
package org.pocketx.knell

import com.squareup.leakcanary.LeakCanary
import timber.log.Timber

class KnellDebugApp : KnellApp() {

    override fun onCreate() {
        Timber.plant(Timber.DebugTree())
        super.onCreate()
        if (LeakCanary.isInAnalyzerProcess(this)) {
            // This process is dedicated to LeakCanary for heap analysis.
            // You should not init your app in this process.
            return
        }
        LeakCanary.install(this)
    }
}
